composer.json

```diff
@@ -43,13 +43,13 @@
             "homepage":"https://github.com/chronon"
         }
     ],
-    "require": {
-        "php": ">=5.6",
-        "cakephp/cakephp": "^3.5.5"
+    "require":{
+        "cakephp/cakephp": "4.x-dev as 4.0.0",
+        "cakephp/plugin-installer": "*"
     },
     "require-dev": {
-        "cakephp/cakephp-codesniffer": "^3.0",
-        "phpunit/phpunit": "^5.7.14|^6.0"
+        "phpunit/phpunit": "^7",
+        "cakephp/cakephp-codesniffer": "dev-master"
```

Comment:
The coding standard has been updated for Cake 4. This needs to be `dev-next`.

Reply:
switched to dev-next